strings/ngram.py

```diff
@@ -18,9 +18,9 @@ def create_ngram(sentence: str, ngram_size: int) -> list:
     ['I ', ' a', 'am', 'm ', ' a', 'an', 'n ', ' N', 'NL', 'LP', 'Pe', 'er']
     """
```

Comment:
Please add a test...
```suggestion
    >>> create_ngram("This is short", 50)
    ???
    """
```

Reply:
Please do not mark this resolved until after the test passes.

Please replace the `???` with `[]`.  Is `[]` the list of ngrams that we expected?

Reply:
Yes, changes done.